### PR TITLE
2013 firefox test fixes

### PR DESCRIPTION
**Pull request by alexlemann**
Work to get selenium tests working in FF as well as phantomjs now that the new Firefox API geckodriver has been released.

There are remaining issues with access control on resource pages and mezzanine that I am unsure how to handle. It seems like the templates are still using page permissions rather than newer hs_access control permissions.

### CODE REVIEW

File: theme/tests/multiplatform.py [hunk 1]
```diff
@@ -98,6 +97,7 @@ def setUp(self):
                     groups=[group]
```
Comment (pkdash) [inline theme/tests/multiplatform.py]: @alexlemann Have you tried using here ```groups=[ ]``` to resolve permission issues?

**Comment by pkdash** [comment]
@alexlemann Do you plan to make any further changes to this PR? Otherwise +1.

File: theme/tests/multiplatform.py (hunk 1 above)
Comment (alexlemann) [inline theme/tests/multiplatform.py]: I have not. I'm looking for input on how to setup the groups and perms. What would effect would leaving groups empty have?

**Comment by alexlemann** [comment]
@pkdash No, it's good to go from my perspective.

File: theme/tests/multiplatform.py (hunk 1 above)
Comment (alexlemann) [inline theme/tests/multiplatform.py]: @pkdash Do you have any additional information for me here?

**Pull request merged**